src/store/index.js

```diff
@@ -108,6 +109,7 @@ export default new Vuex.Store({
         saveErrorLog,
         tourStartBar,
         tokens: { all },
+        invites: { links },
```

Comment:
After removing sdk from state:
```suggestion
        invites,
```